### PR TITLE
Feature/16 fix android browserstack

### PR DESCRIPTION
**Pull request by EmilianoMorghen**
Added Ruby and Bundler optional parameters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ This action build Android project, export .apk file as GitHub artifact, with opt
 
 Tested with Ionic, React Native and native android projects.
 
-## New in 1.2.0
+## New in 1.3.0
```
Comment (francescoben) [inline README.md:7]: @EmilianoMorghen rather than deleting the 1.2 version section, add the 1.3 details above it

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,19 @@
 # Change Log
 
+## [1.3.0] - 2023-30-01
+
+Added optional Ruby and bundler version
+
+### Added
+
+- Optional Ruby version: using the `ruby-version` property you can specify Ruby version you wish to use. If missing latest Ruby version available will be used.
+- Optional bundler version: using the `bundler-version` property you can specify bundler version you wish to use. If missing latest bundler version will be used.
```
Comment (francescoben) [inline CHANGELOG.md:10]: @EmilianoMorghen this new version also add the `fastlane-env` parameter, please add a paragraph about it

File: README.md (hunk 1 above)
Comment (EmilianoMorghen) [reply README.md:7]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (EmilianoMorghen) [reply CHANGELOG.md:10]: done

File: README.md (hunk 1 above)
Comment (EmilianoMorghen) [reply README.md:7]: Done

**Review by francescoben — APPROVED** [review]

**Comment by francescoben** [comment]
closes #12 #16

**Pull request merged**